### PR TITLE
Fixing Apache POI issue - Formula Cell evaluation nullpointer exception

### PR DESCRIPTION
**Pull request by wkk91193**
Fixing Apache POI Formular Cell evaluation null pointer thrown. Known issue in the previous version of data import tool and fixed by [PR](https://github.com/openMF/DataImportTool/pull/34)

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/ImportHandlerUtils.java [hunk 1]
```diff
@@ -63,8 +63,13 @@ public static Long readAsLong(int colIndex, Row row) {
         FormulaEvaluator eval = row.getSheet().getWorkbook().getCreationHelper().createFormulaEvaluator();
         if(c.getCellType() == Cell.CELL_TYPE_FORMULA) {
             if(eval!=null) {
-                CellValue val = eval.evaluate(c);
-                return ((Double) val.getNumberValue()).longValue();
+                CellValue val = null;
```
Comment (ShruthiRajaram) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/ImportHandlerUtils.java:66]: Can you please let us know in which scenario a null pointer exception is thrown.  As there is a null check for  'c and eval', I am not able to figure out  the case when a null pointer exception is thrown  by the statement in try block . This applies for all remaining method changes.

**Review by ShruthiRajaram — CHANGES_REQUESTED** [review]
Create a JIRA issue and raise PR

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/ImportHandlerUtils.java (hunk 1 above)
Comment (wkk91193) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/ImportHandlerUtils.java:66]: @ShruthiRajaram When any excel formula is evaluated to null, a null pointer is thrown fromApache POI API, this is what's being handled.

Comment (wkk91193) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/ImportHandlerUtils.java:66]: A known issue in the previous version of data import tool and fixed by PR in the description

Comment (ShruthiRajaram) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/ImportHandlerUtils.java:66]: Referring https://github.com/openMF/DataImportTool/pull/34

**Review by ShruthiRajaram — APPROVED** [review]
referring , https://github.com/openMF/DataImportTool/pull/34

**Comment by MexinaD** [comment]
@ShruthiRajaram  The formula cell nullpointer exception was causing https://jira.apache.org/jira/browse/FINERACT-660 issue. Does this PR need to have its specific JIRA issue to be merged or i can merge it to solve this crucial issue?

**Pull request merged**